src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -32,14 +32,14 @@
       <_NET50TargetingPackVersion>5.0.0</_NET50TargetingPackVersion>
       <_WindowsDesktop50RuntimePackVersion>5.0.1</_WindowsDesktop50RuntimePackVersion>
       <_WindowsDesktop50TargetingPackVersion>5.0.0</_WindowsDesktop50TargetingPackVersion>
-      <_AspNet50RuntimePackVersion>5.0.1</_AspNet50RuntimePackVersion>
+      <_AspNet50RuntimePackVersion>5.0.2</_AspNet50RuntimePackVersion>
```

Comment:
@marcpopMSFT Were the runtime pack versions supposed to be handled by your change?

Reply:
These are automatic in `release` branches, but that logic isn't in master - I figured it was because master isn't incrementing its patch version at the same cadence as the release branches, so the calculation wouldn't be valid/consistent

Reply:
Makes sense.  We could make it so that we only have to update the 5.0 version in the master branch and then the 3.1 and 2.1 versions would be calculated from that.